packages/swap-sdk-core/src/fiatCurrency.ts

```diff
@@ -0,0 +1,69 @@
+import { Currency } from './currency'
+
+/**
+ * A currency is any fungible financial instrument, including Ether, all ERC20 tokens, and other chain-native currencies
+ */
+export abstract class FiatCurrency {
```

Comment:
Wonder what's this class use for? If it's nothing to do with swap, better not put it into sdk

Reply:
removed. initially i waned to keep similar structure between token and cuurencies cos there similar and i initially made this class because theres a check in the currencyList componennt that uses the 'equals' class. so initially when i just defined the fiat currencies as a maping there would be a runtime error as method equals does not exists on mapping etc... anyways that was why i initially made this class. removed ages ago